examples/MomentoApplication/Program.cs

```diff
@@ -16,7 +17,7 @@ class Program
 
         async static Task Main(string[] args)
         {
-            using SimpleCacheClient client = new SimpleCacheClient(MOMENTO_AUTH_TOKEN, DEFAULT_TTL_SECONDS);
+            using SimpleCacheClient client = new SimpleCacheClient(Configurations.Laptop.Latest, MOMENTO_AUTH_TOKEN, DEFAULT_TTL_SECONDS);
```

Comment:
how does this pass CI?  My expectation is (for better or for worse) that the examples solution file is pinned to the release version of the SDK from nuget.